l10n_ch_base_bank/models/invoice.py

```diff
@@ -121,15 +112,22 @@ def _check_isr(self):
                 invoice._is_isr_reference()
         return True
 
-    @api.model
-    def create(self, vals):
-        """We override create in order to have customer invoices
-        generated by the comercial flow as on change partner is
-        not systemtically call"""
-        type_defined = vals.get('type') or self.env.context.get('type', False)
-        if type_defined == 'out_invoice' and not vals.get('partner_bank_id'):
-            user = self.env.user
-            bank_ids = user.company_id.partner_id.bank_ids
+    def partner_banks_to_show(self):
+        """
+        Extend method from account_payment_partner to add specific
+        logic for switzerland bank payments if base method give false result
```

Comment:
I would reword it like this, as `false` could be interpreted as `wrong`

```suggestion
        logic for switzerland bank payments if base method does not give a result
```